ruma-federation-api/src/user/get_devices/v1.rs

```diff
@@ -0,0 +1,35 @@
+//! [GET /_matrix/federation/v1/user/devices/](https://matrix.org/docs/spec/server_server/r0.1.4#get-matrix-federation-v1-user-devices-userid)
+
+use ruma_api::ruma_api;
+use ruma_client_api::r0::Device;
```

Comment:
The various API crates don't have dependencies on each other. To use this type, first create a `device` module in `ruma-common` and move that type to there (you can make it a separate PR, or a separate commit in this PR). Alternatively, just copy-paste the definition of that type for now.